### PR TITLE
Run tests on pre-push hook

### PR DESCRIPTION
**Pull request by krisgardiner**
#### What does this PR do?
- Runs tests on pre-commit hook

#### Description of Task to be completed?
- Ensuring tests run before each commit increases confidence in code quality/stability.

#### How should this be manually tested?
- yarn run test:all

#### What are the relevant Trello Card stories? [ Trello Card Item ](https://trello.com/b/8x3mo2V5/say-their-names-web)
- N/A

#### Pull Request Type (check one)
- [ ] Feature
- [ ] Bug Fixes
- [x] Chore

#### Developer Checklist
- [x] New code has been scanned for ESlint issues
- [x] I have tested my code locally
- [x] I made sure that there are no other pull requests that solve for this issue
- [x] My code follows the same structure within the guidelines

#### Screenshots (if appropriate)
image

### CODE REVIEW

**Comment by krisgardiner** [comment]
Not sure if we want to set a minimum test coverage to ensure new features are being tested.

**Comment by Easybuoy** [comment]
> Not sure if we want to set a minimum test coverage to ensure new features are being tested.

To do that we would need to have written some amount of tests

File: .huskyrc.json [hunk 1]
```diff
@@ -1,5 +1,5 @@
 {
   "hooks": {
-    "pre-commit": "yarn run lint:fix"
+    "pre-commit": "yarn run test:all"
```
Comment (Easybuoy) [inline .huskyrc.json:3]: Can we have the test run on `pre-push` instead, so we have only listing run on `pre-commit`

File: package.json [hunk 2]
```diff
@@ -23,6 +24,7 @@
     "start": "react-scripts start",
     "build": "react-scripts build",
     "test": "react-scripts test",
+    "test:all": "yarn run lint:fix && CI=true react-scripts test",
```
Comment (Easybuoy) [inline package.json:27]: You can use `yarn test` here instead of `react-scripts test`

**Comment by krisgardiner** [comment]
Added to https://github.com/Say-Their-Name/say-their-names-web/pull/53

**Pull request closed without merging**